README.md

```diff
@@ -23,19 +23,68 @@ English | [简体中文](./README.zh-CN.md)
 
 Dashboard for [Apache APISIX](https://github.com/apache/apisix)
 
-## User Guide
+The goal of Apache APISIX Dashboard project is to enable everyone to quickly experience and learn Apache APISIX, and it still cannot be used directly in a production environment. its function point coverage is always a subset of Apache APISIX, and may lag behind the rapid iteration of Apache APISIX. 
 
-Please refer to [User Guide](./docs/USER_GUIDE.md)
+If you need to use the Dashboard project in a production system, you need to enhance user permissions, communication security, high availability and other advanced features.
+
+## Install
+
+Support multiple ways to install APISIX dashboard
+
+### Docker
+
+Start a runnable version by the following method
+
+- [Deploy With Docker](./docs/deploy-with-docker.md)
+
+### Build From Source Code
+
+To build from source code, first make sure that your `golang` version is 1.13 or greater.
+Also you need to follow the `node` and `yarn` in advance
+
+```
+$ git clone -b v2.0 https://github.com/apache/apisix-dashboard.git && cd apisix-dashboard
+$ make build
+```
+
+Then you can find all files (configuration files, executable files, web static resources) needed to run dashboard in the `./output` directory.
+
+Start by the following command
```

Comment:
Hmmm cc @moonming I have no idea how to describe my feelings...

Reply:
Start by the following commands